Specs/Scene/ShadowMapSpec.js

```diff
@@ -0,0 +1,1072 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/ShadowMap',
+        'Core/BoundingSphere',
+        'Core/BoxGeometry',
+        'Core/Cartesian3',
+        'Core/ComponentDatatype',
+        'Core/Color',
+        'Core/ColorGeometryInstanceAttribute',
+        'Core/defined',
+        'Core/EllipsoidTerrainProvider',
+        'Core/GeometryInstance',
+        'Core/HeadingPitchRange',
+        'Core/HeightmapTerrainData',
+        'Core/JulianDate',
+        'Core/Math',
+        'Core/PixelFormat',
+        'Core/Transforms',
+        'Renderer/Context',
+        'Renderer/Framebuffer',
+        'Renderer/PixelDatatype',
+        'Renderer/Texture',
+        'Renderer/WebGLConstants',
+        'Scene/Camera',
+        'Scene/Globe',
+        'Scene/Model',
+        'Scene/OrthographicFrustum',
+        'Scene/PerInstanceColorAppearance',
+        'Scene/Primitive',
+        'Specs/createScene',
+        'Specs/pollToPromise',
+        'ThirdParty/when'
+    ], function(
+        ShadowMap,
+        BoundingSphere,
+        BoxGeometry,
+        Cartesian3,
+        ComponentDatatype,
+        Color,
+        ColorGeometryInstanceAttribute,
+        defined,
+        EllipsoidTerrainProvider,
+        GeometryInstance,
+        HeadingPitchRange,
+        HeightmapTerrainData,
+        JulianDate,
+        CesiumMath,
+        PixelFormat,
+        Transforms,
+        Context,
+        Framebuffer,
+        PixelDatatype,
+        Texture,
+        WebGLConstants,
+        Camera,
+        Globe,
+        Model,
+        OrthographicFrustum,
+        PerInstanceColorAppearance,
+        Primitive,
+        createScene,
+        pollToPromise,
+        when) {
+    'use strict';
+
+    var scene;
+    var sunShadowMap;
+    var backgroundColor = [0, 0, 0, 255];
+
+    var longitude = -1.31968;
+    var latitude = 0.4101524;
+    var height = 0.0;
+    var boxHeight = 4.0;
+    var floorHeight = -1.0;
+
+    var boxUrl = './Data/Models/Shadows/Box.gltf';
```

Comment:
Are all the test models in the right location now?  At one time, we had Sandcastle examples pointing to the tests or vice versa.